tests/snapshot_iterator.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/syndtr/goleveldb/leveldb"
+)
+
+func main() {
```

Comment:
Is it okay if there is main() function here?

Reply:
Sorry some late.
This code has been changed to verify-tools.